### ISSUE TITLE
When sending bitcoin don't use an unconfirmed transaction as input

### ISSUE THREAD

**Issue by Thoragh** [LedgerHQ/ledger-wallet-chrome#61]
Make it so that when sending bitcoin it uses transactions that have already confirmed as inputs if possible.

Its annoying when creating a transaction with low fee and then create another transaction with higher fee
just to find out that i will have to wait for the first transaction to confirm before the second can confirm.

**Comment by pollastri-pierre** [comment]
Do you have an example, because this is actually what it does. We are consuming the most confirmed inputs first.

**Comment by Thoragh** [comment]
It created transaction 9c42eb15295800d772534526bd8405f8507f830af09bc83bcfd7cfa61c8f8be0 when dfa06c70398b0db1da080541a8ff79e85ac9307cf4ce2b5e5bca2f11466d134a was not confirmed.

**Comment by pollastri-pierre** [comment]
Yes but do you have the list of your UTXO at this moment. Because if you don't have any confirmed UTXO, the application will use an unconfirmed one.

**Comment by Thoragh** [comment]
It could have skipped the unconfirmed input and still have enough.

**Comment by dmcnaught** [comment]
Here is an example from today

https://blockchain.info/tx/3c11559cebb72a93ea5f8bdb3998e2def2cc197119fbd237dc52da40a3518e1c

outputs to 1N5DsWTtPvWt32LWXMTa6cNN3eskNQAuLX and that was used as an input to 

https://blockchain.info/tx/837943a19f027b770123a4c318638dd82486988cc926b84667e7655b1b143489
